pkg/controller/baremetalhost/baremetalhost_controller_test.go

```diff
@@ -124,7 +124,8 @@ func tryReconcile(t *testing.T, r *ReconcileBareMetalHost, host *metalkubev1alph
 			break
 		}
 
-		if !result.Requeue {
+		t.Logf("result: %v", result)
```

Comment:
It's not clear to me if this is a leftover line from debugging, or if you meant to leave it

Reply:
I did mean to leave that, to show the result in the loop. It's only printed to the console if the test fails.

Reply:
OK, thanks